### PR TITLE
[[YUNIKORN-2382] Add supported K8s versions

### PR DESCRIPTION
**Pull request by wilfred-s**
### What is this PR for?
Add supported K8s to YuniKorn release list to the docs

Chinese translation is tool based translation and needs a proper check before committing.

### What type of PR is it?
* [X] - Improvement
* [X] - Documentation

### What is the Jira issue?
* https://issues.apache.org/jira/browse/YUNIKORN-2382

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 应该使用哪个版本？
+
+建议始终运行最新版本的 YuniKorn。新版本定期由 YuniKorn 本身和/或垫片的新功能驱动。目前，功能和修复仅添加到已删除版本的开发分支中。
```
Comment (chia7712) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:27]: `垫片` could be replaced by `shim` I think :)

File: docs/get_started/version.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Which version should be used?
+
+The recommendation is to always run the latest possible version of YuniKorn.
+New releases are made regular driven by new features for YuniKorn itself and or the shim. 
+Features and fixes are currently only added to the development branch from which releases are cut.
```
Comment (chia7712) [inline docs/get_started/version.md:29]: Maybe we can highlight that "there is no patch version" currently. I guess most engineers are aware of semantic versioning :)

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+如果 shim 版本（即 Kubernetes）受多个 YuniKorn 版本支持，我们建议始终使用最新的 YuniKorn 版本。
+早期版本中较旧的构建依赖项可能会阻止您使用较新的填充程序公开的某些新功能。
```
Comment (chia7712) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:30]: `早期版本中较旧的构建依赖项可能会阻止您使用较新的填充程序公开的某些新功能。` could be replaced by `早期版本中较旧的构建依赖项可能会阻止您使用 shim 的某些新功能。`

**Review by chia7712 — COMMENTED** [review]
@wilfred-s I like this new page! a couple comments left. PTAL

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md (hunk 1 above)
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:27]: done

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md (hunk 3 above)
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:30]: done

File: docs/get_started/version.md (hunk 2 above)
Comment (wilfred-s) [reply docs/get_started/version.md:29]: added a line explaining that, in both english and zh-cn

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# 应该使用哪个版本？
+
+建议始终运行最新版本的 YuniKorn。新版本定期由 YuniKorn 本身和/或 shim 片的新功能驱动。目前，功能和修复仅添加到已删除版本的开发分支中。
```
Comment (chia7712) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:27]: `建议始终运行最新版本的 YuniKorn。新版本定期由 YuniKorn 本身和/或 shim 的新功能驱动。目前新功能和修复仅合并到用来做释出的开发分支。`

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+此时该项目仅创建次要版本，没有补丁版本。
+
+如果 shim 版本（即 Kubernetes）受多个 YuniKorn 版本支持，我们建议始终使用最新的 YuniKorn 版本。
+早期版本中较旧的构建依赖项可能会阻止您使用 shim 的某些新功能。
+
+# 每个 YuniKorn 版本的 Kubernetes 版本
```
Comment (chia7712) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:34]: `YuniKorn 支持的 Kubernetes 版本`

**Review by chia7712 — COMMENTED** [review]
@wilfred-s a couple of comments left. PTAL

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+| K8s 版本       | 从版本支持  |  支持结束  |
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:36]: done

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md (hunk 5 above)
Comment (wilfred-s) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/version.md:34]: done

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request closed without merging**